src/main/java/com/patreon/euphrates/Config.java

```diff
@@ -114,10 +115,12 @@ public Table(
     public S3(
                @JsonProperty("bucket") String bucket,
                @JsonProperty("region") String region,
+               @JsonProperty("accessKey") String iamRole,
```

Comment:
Should the Jon property be "iam role" as well?

Reply:
Oops!